spec/connectors_sdk/sharepoint/http_call_wrapper_spec.rb

```diff
@@ -157,20 +157,20 @@ def mock_endpoint(path, data)
   end
 
   context '.source_status' do
-    let(:access_token) { 'access token' }
+    let(:params) { { :access_token => 'access_token' } }
```

Comment:
maybe we could add `:index_permissions` here and verify that `ConnectorsSdk::Office365::Config` gets it